core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthChecker.java

```diff
@@ -105,6 +106,19 @@ static HealthChecker ofCpu(double targetSystemCpuUsage, double targetProcessCpuU
         return new CpuHealthChecker(targetSystemCpuUsage, targetProcessCpuUsage);
     }
 
+    /**
+     * Creates a new instance of {@link HealthChecker}
+     * which reports health based on disk space of specific file path.
+     * Both free disk space must be (inclusive) lower than the specified threshold in order
+     * for the {@link HealthChecker} to report as healthy.
+     *
+     * @param targetFreeDiskSpace Target free disk space in bytes.
```

Comment:
```suggestion
     * @param diskSpacePercentage the target free disk space as a percentage (0 - 1).
```

Reply:
I ammended `targetDiskSpacePercentage` as you suggested below. Please, consider it.